installer/common/detect.sh

```diff
@@ -31,19 +31,19 @@ detect_sparkoperator() {
 
 # Returns the namespace where the Flink operator is installed or an empty string
 detect_flinkoperator() {
-	kubectl get deployment -l app.kubernetes.io/name=flink-operator --all-namespaces --ignore-not-found=true --no-headers | awk -v i=1 -v j=1 'FNR == i {print $j}'
+    kubectl get deployment -l app.kubernetes.io/name=flink-operator --all-namespaces --ignore-not-found=true --no-headers | awk -v i=1 -v j=1 'FNR == i {print $j}'
 }
 
 # Returns true if the Flink operator is configured to watch all namespaces
 detect_flink_watch_all_namespaces() {
-	# Currently awaiting a fix to make the operator config easier to parser in Bash.
-	#local result=$(kubectl get configmap -n lightbend flink-operator-config -o json --ignore-not-found=true | jq '.data.config.limitNameSpace')
-	#if [ -z result ]; then
-	#	result=true
-	#else
-	#	result=false
-	#fi
-	echo true
+  # Currently awaiting a fix to make the operator config easier to parser in Bash.
+  #local result=$(kubectl get configmap -n lightbend flink-operator-config -o json --ignore-not-found=true | jq '.data.config.limitNameSpace')
+  #if [ -z result ]; then
+  #	result=true
+  #else
+  #	result=false
+  #fi
+  echo true
 }
```

Comment:
@lightbend/cloudflow-team is this commented out function already being tracked on a TODO list or on the issue list?